cpp/src/parquet/arrow/reader_internal.cc

```diff
@@ -487,6 +535,8 @@ Status GroupToSchemaField(const GroupNode& node, int16_t max_def_level,
                           const SchemaField* parent, SchemaField* out) {
   if (node.logical_type()->is_list()) {
     return ListToSchemaField(node, max_def_level, max_rep_level, ctx, parent, out);
+  } else if (node.logical_type()->is_map()) {
```

Comment:
It doesn't seem like this handles the backward compatibility case of "MAP_KEY_VALUE" listed in the [spec](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#maps)